accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -429,7 +429,10 @@ disambiguate the extension type declaration with a fixed lookahead.*
 Some errors can be detected immediately from the syntax:
 
 A compile-time error occurs if the extension type declaration declares any
-instance variables.
+instance variables, unless they are `external`.
+
+*An external instance variable is just a convenient notation for an external
+getter and (if not `final`) an external setter. They are allowed.*
```

Comment:
Extremely nit-picky: Consider changing "They" to "Those".
I like the more demonstrative pronoun.

Alternately ", which are both allowed."

Reply:
Done in https://github.com/dart-lang/language/pull/3309.